atom/browser/atom_browser_client.cc

```diff
@@ -174,6 +181,10 @@ void AtomBrowserClient::AppendExtraCommandLineSwitches(
     base::CommandLine* command_line,
     int process_id) {
   std::string process_type = command_line->GetSwitchValueASCII("type");
+
+  command_line->AppendSwitchASCII(switches::kRegisterStandardSchemes,
```

Comment:
I think we don't need to append `kRegisterStandardSchemes` for processes other than renderer process?